core/templates/dev/head/app.js

```diff
@@ -72,8 +72,7 @@ oppia.constant('EVENT_ACTIVE_CARD_CHANGED', 'activeCardChanged');
 // The conditioning on window.GLOBALS.RTE_COMPONENT_SPECS is because, in the
 // Karma tests, this value is undefined.
 oppia.constant(
-  'RTE_COMPONENT_SPECS',
-  window.GLOBALS.RTE_COMPONENT_SPECS ? window.GLOBALS.RTE_COMPONENT_SPECS : {});
+  'RTE_COMPONENT_SPECS', richTextComponents ? richTextComponents : {});
```

Comment:
Actually if we specify this in karma-globals then can we get rid of the ternary operator conditioning?

Reply:
Done.